src/components/chip/chip.classnames.ts

```diff
@@ -0,0 +1,70 @@
+import { ChipSize } from './types'
+import { cva } from 'class-variance-authority'
+
+export const chipBaseVariant = cva('chipBaseClassnames', {
+  variants: {
+    size: {
+      sm: ['px-600', 'py-[3px]', 'text-50', 'gap-x-400'],
+      md: ['px-800', 'py-[5px]', 'text-75', 'gap-x-400'],
+      lg: ['px-1200', 'py-[7px]', 'text-100', 'gap-x-400'],
+    },
+    dismissible: {
+      true: ['cursor-default'],
+      false: ['cursor-pointer'],
+    },
+  },
+  defaultVariants: {
+    size: 'md',
+  },
+})
+
+export const ChipIconSizes: Record<
+  ChipSize,
+  { width: number; height: number }
+> = {
+  sm: {
+    width: 11,
+    height: 18,
+  },
+  md: {
+    width: 13,
+    height: 22,
+  },
+  lg: {
+    width: 14,
+    height: 24,
+  },
+}
+
+export const ChipStandaloneIconSizes: Record<
```

Comment:
@shafin-deriv, When I checked, passing in the size e.g. `sm` `md` or `lg` properly sizes the `icons` try it and see, so that you don't have to do these

Reply:
@jim-deriv currently the iconSize is not accurate. I have discussed this with @mohsen-deriv and we have to keep it like this for the time being until the icon size is fixed.